docs/topics/composite-primary-key.txt

```diff
@@ -176,6 +176,18 @@ This form does not have a form field ``pk`` for the composite primary key:
 Setting the primary composite field ``pk`` as a form field raises an unknown
 field :exc:`.FieldError`.
 
+Composite primary keys in model validation
+==========================================
+
+Since ``pk`` is only a virtual field, the ``exclude`` arguments of
+:ref:`model validation <validating-objects>` methods
+:meth:`.Model.full_clean` and :meth:`.Model.clean_fields` do not recognize
+them. To exclude the composite primary key fields from model validation,
+specify each field individually.
```

Comment:
```suggestion
Since ``pk`` is only a virtual field, including ``pk`` as a field name in the
``exclude`` argument :meth:`.Model.clean_fields` has no effect. To exclude the
composite primary key fields from :ref:`model validation <validating-objects>`,
specify each field individually.
```
Maybe?
As `full_clean` calls `validate_unique`, then I guess passing pk in still has an impact

Reply:
Ah of course!